ibm_db.c

```diff
@@ -132,6 +132,14 @@ static void LogMsg(const char *log_level, const char *message, const char *file_
     }
 }
 
+static void LogUTF8Msg(PyObject *args, const char *fileName)
+{
+    PyObject *argsStr = PyObject_Repr(args);
+    snprintf(messageStr, sizeof(messageStr), "Received arguments: %s", PyUnicode_AsUTF8(argsStr));
+    LogMsg(INFO, messageStr, fileName);
```

Comment:
No need to pass fileName to LogMsg(). It is directly available inside LogMsg() as global variable. Directly use it inside LogMsg() function.